docs/products/kafka/howto/configure-topic-tiered-storage.md

```diff
@@ -1,146 +1,127 @@
--   [Tiered storage enabled for the Aiven for Apache Kafka service](/docs/products/kafka/howto/enable-kafka-tiered-storage).
+Aiven for Apache Kafka® allows you to configure tiered storage and set retention policies for individual topics.
 
-## Configure tiered storage for topics via Aiven Console
+## Prerequisite
 
-1.  Access [Aiven console](https://console.aiven.io/), select your
-    project, and select your Aiven for Apache Kafka service.
-1.  From the left sidebar, select **Topics**.
-1.  You have the option to either add a new topic with tiered
-    storage configuration or modify an existing topic to use tiered
-    storage.
+- [Tiered storage enabled for the Aiven for Apache Kafka service](/docs/products/kafka/howto/enable-kafka-tiered-storage).
 
-### For a new topic
+## Important information about tiered storage
 
-1.  From the **Topics** page, select **Add topic**.
+When you activate tiered storage for a service, any new topics created in that service
+have tiered storage enabled by default.
 
-1.  Set the "Do you want to enable advanced configuration?" option to Yes to enable
-advanced configurations.
+- **Disable tiered storage for new topics**: You can disable tiered storage by setting
+  **Remote storage enable** to false during the creation of a new topic in the topic
+  advanced configurations. Once you activate tiered storage for a topic, you cannot
+  disable it. Contact Aiven support for assistance.
 
-1.  In the **Topic advanced configuration** drop-down, choose
-    `remote_storage_enable`. This action will reveal the **Remote
-    storage enabled** drop-down.
+- **Tiered storage for existing topics**: For existing topics, tiered storage is disabled.
 
-1.  Select **True** to activate tiered storage for the topic.
+## Enable and configure tiered storage for topics
 
-1.  Additionally, you can also set the values for `local_retention_ms`
-    and `local_retention_bytes` using the respective options from the
-    drop-down list.
+<Tabs groupId="setup">
+<TabItem value="Console" label="Console" default>
 
-    :::important
-    If the values for `local_retention_bytes` and `local_retention_ms`
-    are not set, they default to -2 or take the configuration from the
-    service level.
+1. Access the [Aiven Console](https://console.aiven.io/), select your project,
+   and select your Aiven for Apache Kafka service.
+1. Click <ConsoleLabel name="topics" />.
 
-    When set to -2, the retention in local storage will match the total
-    retention. In this scenario, the data segments sent to remote
-    storage are also retained locally. The remote storage will contain
-    older data segments than in the local storage only when the total
-    retention is set to be greater than the local retention.
-    :::
+1. On the **Topics** page, either add a new topic with tiered storage or modify an
+   existing one:
+   - **Add a new topic**: Click **Add topic**.
+   - **Modify an existing topic**: Select the topic to modify, then either:
```

Comment:
Consider removing bold for "Modify an existing topic" coz it might imply to the user they need to find this option in the GUI.

Also, "either" requires "or" so how about replacing "Alternatively" with "or". You can also drop "either" and keep "Alternatively" as is. Up to you.